Makefile

```diff
@@ -133,7 +133,7 @@ ifneq ("$(NO_CROSS_COMPILER)", "")
 else
 	mkdir -p ./bin/$(TARGETOS)_$(TARGETARCH)
 	$(CXX) -o ./bin/$(TARGETOS)_$(TARGETARCH)/syz-executor$(EXE) executor/executor.cc \
-		$(ADDCXXFLAGS) $(CFLAGS) -DGOOS_$(TARGETOS)=1 -DGOARCH_$(TARGETARCH)=1 \
+		$(ADDCXXFLAGS) $(CXXFLAGS) $(LDFLAGS) -DGOOS_$(TARGETOS)=1 -DGOARCH_$(TARGETARCH)=1 \
```

Comment:
Nit: I wonder whether it would make more sense to rename `ADDCXXFLAGS` into `CXXFLAGS` and just drop `CFLAGS` here. 

@dvyukov Maybe also drop ADDCFLAGS from here since it's apparently used nowhere in the build process?

https://github.com/google/syzkaller/blob/40f469132918c1122058244e85c86f7ec6d033aa/tools/syz-make/make.go#L80

```
$ git grep -r ADDCFLAGS
tools/syz-make/make.go:         {"ADDCFLAGS", strings.Join(target.CFlags, " ")},
```

Reply:
Both sounds good to me, @dvyukov do you agree?

Reply:
ADDCXXFLAGS is what we (tools/syz-make) compute and pass, CFLAGS is what user can add on top.
If we leave only 1 of these, either user won't be able to pass custom flags, or we won't be able to pass our flags.

Reply:
Ack. Let's leave as is.